packages/spor-design-tokens/config.ts

```diff
@@ -1,128 +1,126 @@
-import { tokens } from './tokens';
-
+import { tokens } from "./tokens";
 
 export default {
-  "source": ["tokens/**/*.json"],
-  "platforms": {
-    "rn": {
-      "transforms": [
+  source: ["tokens/**/*.json"],
+  platforms: {
+    rn: {
+      transforms: [
         "name/cti/camel",
         "attribute/cti",
         "size/object",
-        "color/css"
+        "color/css",
       ],
-      "files": [
+      files: [
         {
-          "format": "javascript/module",
-          "destination": "react-native/index.js"
+          format: "typescript/cjs-module",
+          destination: "react-native/index.js",
         },
         {
-          "format": "typescript/rn-typings",
-          "destination": "react-native/index.d.ts"
-        }
-      ]
-    },
-    "javascript": {
-      "transforms": [
-        "attribute/cti",
-        "name/cti/pascal",
-        "size/px",
-        "color/hex"
+          format: "typescript/es-module",
+          destination: "react-native/index.mjs",
+        },
+        {
+          format: "typescript/rn-typings",
+          destination: "react-native/index.d.ts",
+        },
       ],
-      "files": [
+    },
+    javascript: {
```

Comment:
Her konfigurerer jeg opp at jeg vil bruke to nye formatterere – typescript/cjs-module og typescript/es-module. Det finnes noen _nesten_ like inkluderte formatterere, som jeg tok utgangspunkt i, men endret litt.

Resultatet er at det nå outputtes to JavaScript-filer – en som støtter CJS (`module.exports`) og en som støtter ESM (`export default`).